docs/source/getting-started/intro-quickstart/index.rst

```diff
@@ -222,7 +222,7 @@ When opening the site, you will see a screen similar to the following:
 
 .. figure:: ../../_static/images/tutorials/introductory/quickstart-tracking/quickstart-our-experiment.png
     :alt: MLflow UI Experiment view page
-    :width: 1024px
+    :width: 90%
```

Comment:
The fixed width pixel assignments create issues for displaying these images on smaller screens. Updated all instances of fixed declared width / sizes for embedded images.

Reply:
Is this change related to this PR?